hibernate-osgi/src/main/java/org/hibernate/osgi/OsgiClassLoader.java

```diff
@@ -212,4 +214,18 @@ public void clear() {
 		resourceCache.clear();
 	}
 
+	public void removeBundle(Bundle bundle) {
```

Comment:
@gnodet, would it be better if we switched to using a `Map` for `bundles`, keyed off of something like `Bundle#getBundleId()`, rather than having to loop through the whole `Set` every time a `BundleEvent` comes through?